src/main/java/org/jabref/logic/formatter/bibtexfields/NormalizeUnicodeFormatter.java

```diff
@@ -0,0 +1,41 @@
+package org.jabref.logic.formatter.bibtexfields;
+
+import java.text.Normalizer;
+import java.util.Objects;
+
+import org.jabref.logic.cleanup.Formatter;
+
+/**
+ * Clean up field values by formatting Unicode values with Normalize Unicode
```

Comment:
Please provide the normal form reference there. The link should be in the issue description. Also use `html` here for linking.